web/main/colors.css

```diff
@@ -78,6 +82,9 @@
     --flat-active-shadow: inset 0 1px 2px oklch(0% 0 0 / 80%);
     --field-color: color-mix(in oklch, var(--neutral-900), var(--neutral-800));
     --field-shadow: inset 0 1px 2px oklch(0% 0 0 / 60%);
+    --link-color: var(--blue-300);
```

Comment:
Lighthouse complains about link color contrast:

```
Links rely on color to be distinguishable.
```

Reply:
Can I ask what page you tested? Because I tried checking `feeds/categories/` with Lighthouse and didn't receive this message. However, I did get it in the story, probably due to the storybook links. Some of the low contrast color buttons at the top right are links. Maybe it's problem that Lighthouse defines